UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -684,9 +678,23 @@ public void RpcBeforeBodyTransfer()
 	}
 
 	[Command]
-	public void CmdSetActiveHand(NamedSlot hand)
+	public void CmdSetActiveHand(uint handID, NamedSlot NamedSlot)
 	{
-		activeHand = hand;
+		if (handID != 0) if (NetworkIdentity.spawned.ContainsKey(handID) == false) return;
```

Comment:
```suggestion
		if (handID == 0 || NetworkIdentity.spawned.ContainsKey(handID) == false) return;
```
Did a double-take at this line.  Do you want an `OR` here instead?

Reply:
if zero set to null, if not zero make sure it is in spawn list so it's &&